camel/embeddings/open_source_embeddings.py

```diff
@@ -0,0 +1,55 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from typing import Any, List
+
+from sentence_transformers import SentenceTransformer
+
+from camel.embeddings import BaseEmbedding
+
+
+class E5LargeV2Embedding(BaseEmbedding[str]):
```

Comment:
Why only support `E5LargeV2Embedding`? Why not support all the embedding methods in `SentenceTransformer` if the only change is the model name?